src/components/pages/Map/MapPage.jsx

```diff
@@ -56,6 +67,10 @@ export default class MapPage extends React.Component {
 MapPage.propTypes = {
   layersActive: React.PropTypes.array,
   setMapParams: React.PropTypes.func,
+  setScope: React.PropTypes.func,
```

Comment:
What do you think if we start ordering the propTypes by category? I mean sort them by state, action and selector. So whenever I look into the propTypes I will know what kind of properties it has.